mayastor/src/subsys/nvmf/subsystem.rs

```diff
@@ -224,13 +228,27 @@ impl NvmfSubsystem {
                 .to_string()
         }
     }
+
     /// allow any host to connect to the subsystem
     pub fn allow_any(&self, enable: bool) {
         unsafe {
             spdk_nvmf_subsystem_set_allow_any_host(self.0.as_ptr(), enable)
         };
     }
 
+    /// enable Asymmetric Namespace Access (ANA) reporting
+    pub fn set_ana_reporting(&self, enable: bool) -> Result<(), Error> {
+        unsafe {
+            spdk_nvmf_subsystem_set_ana_reporting(self.0.as_ptr(), enable)
+        }
+        .to_result(|e| Error::Subsystem {
+            source: Errno::from_i32(e),
+            nqn: self.get_nqn(),
+            msg: "failed to set ANA reporting".into(),
```

Comment:
should you capture the enable flag too, eg format!("failed.... {}, enable)?

Reply:
Yes, good point, fixed.